build.gradle

```diff
@@ -713,6 +713,17 @@ subprojects {
       exclude 'junit-platform.properties'
     }
 
+    configurations {
+      tests {
```

Comment:
:bulb: Mental note to myself: configuration name `tests` should be changed to something more appropriate.